app/Http/Controllers/UserController.php

```diff
@@ -36,15 +47,36 @@ public function update(Request $request, User $user)
         $validatedData = $request->validate([
             'name' => 'required|string|max:255',
             'email' => 'required|string|email|max:255|unique:users,email,' . $user->id,
+            'profile_picture' => 'nullable|image|mimes:jpeg,png,jpg,gif,svg|max:2048',
         ]);
 
+        // Check if the "Delete Profile Picture" checkbox is checked
+        if ($request->has('delete_profile_picture')) {
+            // Delete profile picture from storage
+            $user->clearMediaCollection('profile-pictures');
+        }
+
         $user->update($validatedData);
 
-        Notification::success('User updated successfully', true);
+        // Upload new profile picture if provided
+        if ($request->hasFile('profile_picture')) {
+            if ($user->getFirstMedia('profile-pictures')) {
+                $user->clearMediaCollection('profile-pictures');
+            }
 
-        return redirect()->route('users.index');
+            $user->addMediaFromRequest('profile_picture')
+                ->toMediaCollection('profile-pictures');
+        }
+
+        return redirect()->route('users.index')->with('success', 'User updated successfully');
```

Comment:
```suggestion
        Notification::success('User updated successfully', true);

        return redirect()->route('users.index');
```

kita udh pake Notification system yg baru, jd yg sebelum nya ini jgn di hapus